fix(android): prevent high cpu usage in Android Studio

garymathews opened the pull request:
- Disable specific _Inspections_ causing high CPU utilization

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27825)

garymathews committed:
fix(android): prevent high cpu usage in Android Studio

build commented:
<!--
  0 failure: 
  0 warning: 
  3 messages
  
  DangerID: danger-id-Danger;
-->



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :floppy_disk: <a href="https://jenkins.appcelerator.org/job/titanium-sdk/job/titanium_mobile/job/PR-11571/2/artifact/dist/mobilesdk-9.1.0.v20200325141608-osx.zip">Here's the generated SDK zipfile</a>.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: All tests are passing
Nice one! All 105 tests are passing.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>:fist: The commits in this PR match our conventions! Feel free to Rebase and Merge this PR when ready.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b77238f81ed350d8fb2b89ee18858f559bcdac85
</p>

jquick-axway commented:
@garymathews, Android Studio still slams my CPU for about 15-20 minutes with this change.

I sent you an updated inspections XML which makes it super fast for me. Turns out the most expensive inspections were under the "Probably bugs" section. I disabled most of those.

garymathews committed:
fix(android): include additional inspections from jquick

sgtcoolguy commented:
Given this is an internal dev thing, can we merge it in? Does this work well for both @jquick-axway  and @garymathews ?

jquick-axway commented:
@garymathews, we should add the following to our "titanium_mobile/android/.gitignore" file so that we can easily commit changes to this XML file in the future.
```
!/.idea/inspectionProfiles/
```

garymathews committed:
fix(android): exclude inspection profile from git

jquick-axway committed:
chore(android): moved gitignore for inspectionsProfiles

- Fixed gitignore setting for "inspectionProfiles". Wasn't working in previous commit.

jquick-axway committed:
Merge branch 'master' into TIMOB-27825
